### ISSUE TITLE
Oracle and the GRANT statement

### ISSUE THREAD

**Issue by jdonnerstag** [goldmansachs/obevo#141]
According to the docs, Obevo generates statements like the following.
        GRANT SELECT ON TableA TO GROUP RO_GROUP1
        GRANT INSERT, SELECT, UPDATE, DELETE ON TableA TO USER SUPERUSER
    Note the "TO GROUP .." and "TO USER ..".
    However in our legacy scripts GROUP or USER is missing / not required by ORACLE
        GRANT SELECT ON TableA TO RO_GROUP1
        GRANT INSERT, SELECT, UPDATE, DELETE ON TableA TO SUPERUSER
In fact looking at https://docs.oracle.com/cd/B19306_01/server.102/b14200/statements_9013.htm it doesn't seem to be allowed in Oracle

In the Oracle related Obevo source code, I found that Obevo returns "" and thus generated the correct code. Hence, for Oracle it doesn't matter whether I user "user" or "group" in the system-config.xml

How can I use roles, instead of user?  E.g. v_connect is a ROLE
        'GRANT SELECT ON '||R.view_name||' TO ' || v_connect;
Only if Obevo does not create the users, I could define a user even though it is a role. But I don't know whether this misuse of user doesn't create a problem, because users are used somewhere else.

/Juergen

**Comment by shantstepanian** [comment]
Regarding the code generation - as you noted, the Obevo Oracle implementation just returns "" instead of any modifier for the grantee types. This is somewhat counter to the doc example in the permission schemes page. I will clarify the documentation on this point.

Regarding whether to use groups or users - you are correct that in the permission scheme, it technically won't matter which you use, given how Oracle requires grants to be specified. You should go w/ group anyway for consistency.

Per comments in the other issue #140, we currently do not create groups/users in Obevo for Oracle, though we could add this in the future. For now, we assume that the groups/users are already created before calling Obevo.

In terms of whether this causes issues - for now, this shouldn't create a problem. We may in the future try to validate the group/users defined (i.e. check if they actually exist), so I'd suggest getting it correct now.

In summary - go w/ the permission schemas and specify the group attribute to create permissions for roles

**Comment by shantstepanian** [comment]
Hi, please let us know if you have further questions on this, or if we can close the ticket.

We will update the documentation (per my first reply) in the next release, per ticket #150
